### ISSUE TITLE
channel index on left doesn't refresh on update

### ISSUE THREAD

**Issue by lauherkcambria** [robert-paul/slack-clone#1]
the channels index on the left doesn't update when adding a new channel

**Comment by lauherkcambria** [comment]
@robert-paul , nice demo!!! 👍  :)  one small thing i found, you can fix this issue by adding the line: 

 ` @chatroom_user = @chatroom.chatroom_users.where(user_id: current_user.id).first_or_create`

in the create method after you save the chatroom. 

i was going to make a PR for this, but my user, @lauherkCambria, doesn't have permission.

**Comment by robert-paul** [comment]
@lauherkcambria Thanks for the compliment and creating an issue on this :)

I took some time in thinking about what the problem you are describing is, and I think I know what you are mentioning. 

When a user creates a new channel, that chatroom channel is created however the user who created it is not automatically a member of that chatroom channel. 

Yes, this needs to be fixed. For expediency I'll merge it now but I've also now made you a contributor, so if you see anything else come up feel free to make a PR.

Thanks again :) 
 - Rob
